### PR TITLE
Markdown linter and spelling corrections.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@rootfs @gman0 PTAL :)

File: docs/deploy-cephfs.md [hunk 1]
```diff
@@ -69,33 +73,34 @@ YAML manifests are located in `deploy/cephfs/kubernetes`.
...
 
 Deploys a daemon set with two containers: CSI driver-registrar and the CSI CephFS driver.
 
 ## Verifying the deployment in Kubernetes
 
-After successfuly completing the steps above, you should see output similar to this:
+After successfully completing the steps above, you should see output similar to this:
+
 ```bash
 $ kubectl get all
```
Comment (Madhu-1) [inline docs/deploy-cephfs.md]: need to remove `$` here also

Comment (humblec) [reply docs/deploy-cephfs.md]: No @Madhu-1 . The `$` has to be removed only when there is no output

**Comment by rootfs** [comment]
thanks @humblec 

there is a mention of kubernetes 1.11, can you make it 1.13 for this branch?

**Comment by humblec** [comment]
> thanks @humblec
> 
> there is a mention of kubernetes 1.11, can you make it 1.13 for this branch?

Sure. I am updating the PR.

**Comment by humblec** [comment]
@rootfs done. ptal.  thanks !

**Pull request merged**